skimage/measure/_regionprops.py

```diff
@@ -835,6 +848,137 @@ def regionprops(label_image, intensity_image=None, cache=True):
     return regions
 
 
+def euler_number(image, neighbourhood=8):
```

Comment:
Hey @yg42,
I meant both, actually :) As @jni said, the other functions we have are referring to this as `connectivity`.
Now, if I recall correctly, in this scenario `neighborhood` and `connectivity` are equivalent... please help me here @jni :)

Reply:
I totally understand this would be better to use connectivity, in agreement with an overall harmonization of the parameters. After hanging around on the code of different function, I will change this parameter.